pkgs/applications/networking/instant-messengers/gurk-rs/default.nix

```
@@ -11,29 +11,22 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "gurk-rs";
-  version = "0.5.1";
+  version = "0.5.2";
 
   src = fetchFromGitHub {
     owner = "boxdot";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-g0V6FPkCpIEWx+/kDG9+0NtlCVj6jc1gbkkzOSl/lAo=";
+    hash = "sha256-ZVpI60pZZCLRnKdC80P8f63gE0+Vi1lelhyFPAhpHyU=";
   };
 
   postPatch = ''
     rm .cargo/config.toml
   '';
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "libsignal-protocol-0.1.0" = "sha256-4aHINlpVAqVTtm7npwXQRutZUmIxYgkhXhApg7jSM4M=";
-      "libsignal-service-0.1.0" = "sha256-AOGw76A9R2qH3hc7B+MBE3okzW8b5LTZdepzUDOv9lM=";
-      "curve25519-dalek-4.1.3" = "sha256-bPh7eEgcZnq9C3wmSnnYv0C4aAP+7pnwk9Io29GrI4A=";
-      "presage-0.6.2" = "sha256-t9t8ecPtefI/jYlk+Ul8WdgH26VJIkfMptbKxprekS0=";
-      "qr2term-0.3.1" = "sha256-U8YLouVZTtDwsvzZiO6YB4Pe75RXGkZXOxHCQcCOyT8=";
-    };
-  };
+  useFetchCargoVendor = true;
+
+  cargoHash = "sha256-YeRrd2ybCTVAu7EyAZg82eqUP9wvsA8GArdn0oY1eMg=";
```

```suggestion
  cargoHash = "sha256-jTZ2wJPXj3nU7GVTfne64eSra+JuKhNryCtRZMKOE44=";
```
Getting hash mismatch from nixpkgs-reviews.

---

Hmm... after this change, my device makes following error

```console
> nix build .#gurk-rs                                                                                   
error: hash mismatch in fixed-output derivation '/nix/store/r0zm2wwh4q3l1rj9jaj8fcgfq0qg2kn8-gurk-rs-0.5.2-vendor-staging.drv':
         specified: sha256-jTZ2wJPXj3nU7GVTfne64eSra+JuKhNryCtRZMKOE44=
            got:    sha256-YeRrd2ybCTVAu7EyAZg82eqUP9wvsA8GArdn0oY1eMg=
error: 1 dependencies of derivation '/nix/store/g22vkfpbqhyhmksdn4af54j5a8k31cl9-gurk-rs-0.5.2-vendor.drv' failed to build
error: 1 dependencies of derivation '/nix/store/4jcn8nscxrfr6hblszm0zrjf1fmvkbs0-gurk-rs-0.5.2.drv' failed to build
```

---

> Hmm... after this change, my device makes following error
> 
> ```
> > nix build .#gurk-rs                                                                                   
> error: hash mismatch in fixed-output derivation '/nix/store/r0zm2wwh4q3l1rj9jaj8fcgfq0qg2kn8-gurk-rs-0.5.2-vendor-staging.drv':
>          specified: sha256-jTZ2wJPXj3nU7GVTfne64eSra+JuKhNryCtRZMKOE44=
>             got:    sha256-YeRrd2ybCTVAu7EyAZg82eqUP9wvsA8GArdn0oY1eMg=
> error: 1 dependencies of derivation '/nix/store/g22vkfpbqhyhmksdn4af54j5a8k31cl9-gurk-rs-0.5.2-vendor.drv' failed to build
> error: 1 dependencies of derivation '/nix/store/4jcn8nscxrfr6hblszm0zrjf1fmvkbs0-gurk-rs-0.5.2.drv' failed to build
> ```

Well, that's super weird... I was wondering if something changed upstream in past few days but didn't seem like it... but, even more confusing if you're seeing a different hash than on my end. I had 4 different devices build it, too.

---

Thanks for sharing. :pray:  I have adding rebased commit as https://github.com/NixOS/nixpkgs/commit/9e335902fb3efcfbd279680aabcfdaa2005da342, now it does not make error for now.

```console
> nix build .#gurk-rs
> ./result/bin/gurk --version
gurk 0.5.2-dev
```